src/components/Typography/TypographyBase/__test__/TypographyBase.test.tsx

```diff
@@ -1,17 +1,17 @@
-import TyphographyBase, { TypographyBaseProps } from "..";
-
-import { render, screen } from "@testing-library/react";
-
-describe("<TypographyBase />", () => {
-    const props: TypographyBaseProps = {
-        children: "Test",
-    };
-
-    beforeEach(() => {
-        render(<TyphographyBase {...props} />);
-    });
-
-    it("should exist children", () => {
-        expect(screen.getByText("Test")).toBeInTheDocument();
-    });
-});
+import TyphographyBase, { TypographyBaseProps } from "..";
+
+import { render, screen } from "@testing-library/react";
+
+describe("<TypographyBase />", () => {
+    const props: TypographyBaseProps = {
+        children: "Test",
+    };
+
+    beforeEach(() => {
+        render(<TyphographyBase {...props} />);
+    });
+
+    it("should exist children", () => {
+        expect(screen.getByText("Test")).toBeInTheDocument();
+    });
+});
```

Comment:
![image](https://user-images.githubusercontent.com/26225039/140066354-aa78d0a2-aa25-42a1-995f-c37849864e5c.png)
Because I don't think we're testing for the theme style